next/content/docs/guides/vercel.md

```diff
@@ -1,73 +1,47 @@
 ---
-title: Connect a Next.js application to Neon
+title: Integrate Neon with Vercel
 enableTableOfContents: true
-redirectFrom:
-  - /docs/quickstart/vercel
-  - /docs/integrations/vercel
+isDraft: true
 ---
 
-Next.js by Vercel is an open-source web development framework that enables React-based web applications. This topic describes how to create a Neon project and access it from a Next.js application.
+This guide describes how to integrate Neon with Vercel using the Neon integration from the Vercel marketplace. The integration automatically configures the required environment variables in your Vercel project.
 
-To create a Neon project and access it from a Next.js application:
+## What the Neon integration does
 
-1. [Create a Neon project](#create-a-neon-project)
-2. [Create a Next.js project](#create-a-nextjs-project)
-3. [Add a PostgreSQL client to your app](#add-a-postgresql-client-to-your-app)
-4. [Add your Neon connection details](#add-your-neon-connection-details)
-5. [Connect to the Neon database](#connect-to-the-neon-database)
+TBD
 
-## Create a Neon project
+## Prerequisites
 
-If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+- An existing Neon project. If you do not have a Neon project, refer to the [Setting up a project](../../get-started-with-neon/setting-up-a-project/) to get started.
```

Comment:
this isn't necessary, if the user does not has a Neon project they can create one while installing the neon integration in vercel

Reply:
removed.